packages/wallets/src/evm/wallets/device-wallet.ts

```diff
@@ -26,9 +26,10 @@ export class DeviceBrowserWallet extends AbstractBrowserWallet<
 > {
   connector?: TWConnector;
   #walletImpl?: DeviceWalletImpl;
-  static id = "deviceWallet" as const;
```

Comment:
why?

Reply:
We need to overwrite that property in the wallets that extend from those classes since that id is what we use to store the "lastConnectedWallet"

Reply:
We were using the class name as the "lastConnectedWallet" but class names get minified in production builds which was the issue @MananTank was having

Reply:
that technical constraint makes sense. That said this makes me think we're probably doing something wrong somewhere else.